e2e/package-lock.json

```diff
@@ -0,0 +1,6235 @@
+{
```

Comment:
lets get rid of lockfiles for examples and tests. Create `.npmrc` file with `package-lock=false`